import { errorHandler } from "../utils/errorHandler";

export const storingFeedback = (req, res) => {
    try {
    } catch (error) {
        errorHandler(error, res);
    }
};
